scripts/helmcharts/openreplay/charts/utilities/templates/efs-cron.yaml

```diff
@@ -25,7 +25,6 @@ spec:
           restartPolicy: Never
           containers:
           - name: efs-cleaner
-            image: alpine
             image: "{{ tpl .Values.efsCleaner.image.repository . }}:{{ .Values.efsCleaner.image.tag | default .Chart.AppVersion }}"
```

Comment:
We don't have any internal alpine releases. So keep the image as upstream alpine, please.

Reply:
I'm not sure what you're asking me to do here - the image is defined here:
https://github.com/openreplay/openreplay/blob/4872f1f8355e05c2904df90538d9a79700a2a040/scripts/helmcharts/openreplay/charts/utilities/values.yaml#L9

Reply:
ah, that's remnants of some unmerged POCs. I think it's safe to remove the entire image block, as it is not used.

Reply:
https://github.com/openreplay/openreplay/blob/main/scripts/helmcharts/openreplay/charts/utilities/values.yaml#L8-L11

Reply:
Roger that, updated